docs/topics/practices.rst

```diff
@@ -34,15 +34,16 @@ Here's an example showing how to run a single spider with it.
         # Your spider definition
         ...
 
-    process = CrawlerProcess({
-        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)'
+    process = CrawlerProcess(settings={
+        'FEED_FORMAT':'json',
+        'FEED_URI':'items.json'
```

Comment:
a nitpick: could you please add whitespace after `:`, as per pep8?

Reply:
sure